css/New Evaluation.css

```diff
@@ -0,0 +1,46 @@
+body{
+    box-sizing: border-box;
+    margin-left:15%; 
+    margin-right:15%;
+}
+
+ul{
+  list-style-type: none;
+}
+
+li {
+    width:15%;
+}
+
+li{
+    
+}
+
+#OOP li{
+    float: left;
+}
+
+#OOP ul::after{
+  content: "";
+  display: table;
+  clear: both;
+}
+
+#OOP {
```

Comment:
It is a good advice to try to keep this structure clean and intuitive; it makes a lot more sense to start with high level elements and go towards lower levels ones as that gives your code a structure. For example if you have an element that contains an `ul` tag that contains multiple `li` tags I would expect to have a systematic order for definitions:
- first the component
- next the intermediate tags (`ul`)
- the low level tags at the end (`li` in this case)